components/eamxx/src/physics/mam/srf_emission.hpp

```diff
@@ -0,0 +1,138 @@
+#ifndef SRF_EMISSION_HPP
```

Comment:
Could you briefly elaborate the relationship between eamxx_mam_srf_and_online_emissions_process_interface, srf_emission, and srf_emission_impl?

Reply:
`eamxx_mam_srf_and_online_emissions_process_interface` is the overall interface for reading all surface emissions. The code to set up and read those emissions is in `srf_emission` (just the headers), and srf_emission_impl` (body of the functions).